eslzArm/eslzArm.json

```diff
@@ -2704,12 +2709,76 @@
                     "topLevelManagementGroupPrefix": {
                         "value": "[parameters('enterpriseScaleCompanyPrefix')]"
                     },
+                    "logAnalyticsResourceId": {
+                        "value": "[variables('platformResourceIds').logAnalyticsResourceId]"
+                    },
                     "enforcementMode": {
                         "value": "Default"
                     }
                 }
             }
         },
+        {
+            // Assigning Log Analytics Contributor RBAC Role for SQL Auditing Assignment on Management Subscription
+            "condition": "[equals(parameters('enableSqlAudit'), 'Yes')]",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2019-10-01",
+            "name": "[variables('deploymentNames').sqlAuditRoleAssignmentDeploymentName1]",
+            "location": "[deployment().location]",
+            "subscriptionId": "[variables('singleVsDedicatedMgmtSub')]",
+            "dependsOn": [
+                "[variables('deploymentNames').sqlAuditPolicyDeploymentName]"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "contentVersion": "1.0.0.0",
+                    "uri": "[variables('deploymentUris').roleAssignments]"
+                },
+                "parameters": {
+                    "topLevelManagementGroupPrefix": {
+                        "value": "[parameters('enterpriseScaleCompanyPrefix')]"
+                    },
+                    "principalId": {
+                        "value": "[if(equals(parameters('enableSqlAudit'), 'Yes'), reference(variables('deploymentNames').sqlAuditPolicyDeploymentName).outputs.principalId.value, 'na')]"
+                    },
+                    "roleDefinitionId": {
+                        "value": "92aaf0da-9dab-42b6-94a3-d43ce8d16293"
+                    }
+                }
+            }
+        },
+        		{
+            // Assigning SQL Security Manager RBAC Role for SQL Auditing Assignment on Management Subscription 
+			// ***possibly an overkill***
```

Comment:
remove this

Reply:
the comment or the role assignment?

Reply:
understood, I will remove the "possibly an overkill" comment

Reply:
comment "possibly an overkill" removed